src/services/helpers/formValidationRules.js

```diff
@@ -0,0 +1,36 @@
+export const validate = (values) => {
+  let errors = {};
+  if (values.title.length === 0) {
+    errors.title = "Title is required";
+  }
+  if (values.title.length > 0 && values.title.length < 3) {
+    errors.title = "Title has to have at least 3 letters";
+  }
+  if (values.description.length === 0) {
+    errors.description = "Description is required";
+  }
+  if (values.description.length > 0 && values.description.length < 8) {
+    errors.description = "Title has to have at least 8 letters";
+  }
+  if (values.category === "select") {
```

Comment:
hardcoded value, what category will change after some time, use some king of single value

Reply:
I am trying to validate this with yup lib however I cannot find any way to validate that kind of custom input.